src/views/studio/lib/fetchers.js

```diff
@@ -0,0 +1,32 @@
+const ITEM_LIMIT = 4;
+
+const infoFetcher = studioId => fetch(`${process.env.API_HOST}/studios/${studioId}`)
+    .then(d => d.json());
```

Comment:
I can change it to `request`. This area of the code will need some big modifications. I want to formalize the way we handle fetch errors in a better way, although it might be worth just finding a library that does it already. We'll need to customize it since our endpoints tend to return 200s with error messages...

Reply:
Convention is to use `res`/`resp` for `response`.

btw: is Scratch website officially dropping IE9 support? iirc IE8 is dropped in 2015 but not IE9